### PR TITLE
feat(gtfs-archiver) : added view table specific for http error codes

### PR DESCRIPTION
**Pull request by Nkdiaz**
Added a table to view to aid in RT-Archiver analysis. This table selects all the log messages that have a generated HTTP code error

### CODE REVIEW

File: airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: |
 fields:
   calitp_itp_id: Feed ITP ID.
   calitp_url_number: Feed URL number.
-  textPayload: Error message that contains feed key, header length, download url, and error response.
+  token_cleaned_textPayload: Error message that contains feed key, header length, download url, and error response. API Tokens are replaced in the string with API_TOKEN
```
Comment (evansiroky) [inline airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql:11]: Let's try to be consistent with what kind of case is being used.

File: airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql [hunk 2]
```diff
@@ -58,7 +57,7 @@ WITH
 SELECT
   calitp_itp_id,
   calitp_url_number,
-  textPayload,
+  token_cleaned_textpayload,
```
Comment (evansiroky) [inline airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql:60]: Let's try to be consistent with what kind of case is being used. In this instance, I would expect something like:

```suggestion
  token_cleaned_text_payload,
```

**Review by evansiroky — COMMENTED** [review]
Just a comment about consistent casing.

File: airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+operator: operators.SqlToWarehouseOperator
+dst_table_name: "views.gtfs_rt_extraction_html_errors"
```
Comment (lauriemerrell) [inline airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql:3]: Do we want to call this `http_errors` rather than `html_errors`?

**Review by lauriemerrell — CHANGES_REQUESTED** [review]

File: airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql (hunk 3 above)
Comment (Nkdiaz) [reply airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql:3]: ahh good call

File: airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql (hunk 1 above)
Comment (Nkdiaz) [reply airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql:11]: in the gtfs_archiver it is "textPayload".. so would you prefer "textpayload" or "text_payload"

Comment (Nkdiaz) [reply airflow/dags/rt_views/gtfs_rt_extraction_html_errors.sql]: changed it to text_payload

**Comment by holly-g** [comment]
Rendered obsolete by #1463

**Pull request closed without merging**